src/components/security_manager/src/security_manager_impl.cc

```diff
@@ -216,9 +318,21 @@ void SecurityManagerImpl::StartHandshake(uint32_t connection_key) {
   }
 }
 
-bool SecurityManagerImpl::IsCertificateUpdateRequired() {
+bool SecurityManagerImpl::IsCertificateUpdateRequired(
+    const uint32_t connection_key) {
   LOG4CXX_AUTO_TRACE(logger_);
-  return crypto_manager_->IsCertificateUpdateRequired();
+  security_manager::SSLContext* ssl_context =
+      CreateSSLContext(connection_key, kUseExisting);
+  DCHECK(ssl_context);
```

Comment:
@AByzhynar in release this DCHEK  will be removed, use DCHECK_OR_RETURN

Reply:
@LuxoftAKutsan Changed in eb15dd852a0ea418bfa444b74c8f15d792e254ad